### PR TITLE
S3t display recipe cards

### PR DESCRIPTION
**Pull request by georginaw**
Sorted out the formatting of the displayed recipes. Created class with methods to validate the recipe coming back from the API and send an html friendly script to display recipes.

### CODE REVIEW

File: src/Ingredients/DisplayRecipes.php [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+        foreach ($recipes as $recipe) {
+            foreach ($recipe as $recipeItem) {
+                $recipeItem = strval($recipeItem);
+                $recipeItem = preg_replace('/[[:^print:]]/', '', $recipeItem);
+                $newRecipeItem = htmlentities($recipeItem);
+                if ($newRecipeItem != $recipeItem) {
+                    var_dump($newRecipeItem);
+                    var_dump($recipeItem);
+                    return false;
```
Comment (avocadobelly) [inline src/Ingredients/DisplayRecipes.php:41]: get rid of the vardumps

File: src/Ingredients/DisplayRecipes.php [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                        <div class="col-10 col-lg-8">
+                            <h5 class="card-title">' . $recipe->title . '</h5>
+                            <p class="card-text">Ingredients: ' . $recipe->ingredients . '</p>
+                            <a id="linkToRecipeBtn" href="' . $recipe->href . '" class="btn btn-primary float-right">View recipe</a>
+                        </div>
+                    </div>';
+          }
+
+          $recipeCard .= '</div>';
+
```
Comment (avocadobelly) [inline src/Ingredients/DisplayRecipes.php:78]: remove whitespace

File: src/Ingredients/DisplayRecipes.php [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+          return $recipeCard;
+      }
+    }
+
+
```
Comment (avocadobelly) [inline src/Ingredients/DisplayRecipes.php]: whitespace

**Review by avocadobelly — CHANGES_REQUESTED** [review]
just minor chsnges

File: src/Ingredients/DisplayRecipes.php (hunk 1 above)
Comment (Noctiscreo) [reply src/Ingredients/DisplayRecipes.php:41]: done

Comment (Noctiscreo) [reply src/Ingredients/DisplayRecipes.php:41]: done

Comment (Noctiscreo) [reply src/Ingredients/DisplayRecipes.php:41]: (this is georgina by the way...)

File: src/Ingredients/DisplayRecipes.php (hunk 2 above)
Comment (Noctiscreo) [reply src/Ingredients/DisplayRecipes.php:78]: done (georgie)

File: src/Ingredients/DisplayRecipes.php (hunk 3 above)
Comment (Noctiscreo) [reply src/Ingredients/DisplayRecipes.php]: done (georgie)

**Pull request merged**